ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-telemetry-postcard.j2

```diff
@@ -0,0 +1,72 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - monitor telemetry postcard #}
+{% if monitor_telemetry_postcard is arista.avd.defined %}
+!
+monitor telemetry postcard policy
+{%     if monitor_telemetry_postcard.disabled is arista.avd.defined(false) %}
+   no disabled
+{%     endif %}
+{%     if monitor_telemetry_postcard.ingress is arista.avd.defined %}
```

Comment:
Please improve this section to avoid checking the same variables many times. Normally we use a temporary variable to build the CLI. In this case you could build the common part first (after checking the two common fields are set) and then test for version 2 and concatenate like `{% set tmp_cli = tmp_cli ~ " version 2" %}`. Please check out other templates, since this is a very common pattern.

Reply:
Fixed.